plugins/continuous-toolbox/src/ContinuousMetricsFlyout.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Overrides metrics to add extra padding, if needed to the
```

Comment:
Random note: `@fileoverview`s aren't required by the style guide so I've started leaving them out if they are redundant with the class docs. I got tired of trying to figure out how to same the same sentence twice and they aren't adding value so feel free to do the same (unless this is a Blockly-specific rule I've missed, then, oops)